pages/src/common/helpers.less

```diff
@@ -16,16 +16,17 @@
 }
 
 .hide {
+  display: none !important;
```

Comment:
isn't this enough and we need an additional condition for @xs?

Reply:
Yep, it' s a separate `hide` vis `hide-xs` class definition. A little bit of un-obvious LESS ... ))